src/WaCore.Crud.Data.Ef/WacListDataRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+using WaCore.Crud.Contracts.Data;
+using WaCore.Crud.Dtos.Filters;
+using WaCore.Data;
+using WaCore.Data.Repositories.Base;
+
+namespace WaCore.Crud.Data.Ef
+{
+    public abstract class WacListDataRepository<TEntity, TDbContext, TFilter> : WacRepository<TEntity, TDbContext>, IWacListDataRepository<TEntity, TFilter>
+        where TEntity : class
+        where TDbContext : WacDbContext
+        where TFilter : WacFilter
```

Comment:
We can use here IWacFilter interface and DbContext

Reply:
And here again: I already changed this (including other changes), and also will have it in that other PR